pkg/services/cloudmigration/model.go

```diff
@@ -101,6 +102,20 @@ const (
 	ItemStatusPending ItemStatus = "PENDING"
 )
 
+type ResourceErrorCode string
+
+const (
```

Comment:
Do we need to define these constants 3 times? I am just thinking whether adding a new error code will be cumbersome 🤔